src/python_interpreter/mod.rs

```diff
@@ -48,15 +49,33 @@ fn windows_interpreter_no_build(
 
     // There can be 32-bit installations on a 64-bit machine, but we can't link
     // those for 64-bit targets
-    if pointer_width != target_width {
+    if (platform == "win32" && target.pointer_width() == 64)
+        || (platform != "win32" && target.pointer_width() == 32)
+    {
+        eprintln!(
+            "👽 {major}.{minor} reports {platform}, which does not match the target architecture. Skipping."
+        );
+        return true;
+    }
+
+    // Skip mismatching architectures
+    if (platform == "win-arm64" && target.target_arch() != Arch::Aarch64)
+        || (platform == "win-amd64" && target.target_arch() != Arch::X86_64)
```

Comment:
NB: Check what the current default AMD64/x86_64 architecture label would be on Windows. I assume X86_64.

A safer option might be to change the second condition to 

`platform == "win-amd64" && target.target_arch() == Arch::Aarch64`